CHANGELOG.txt

```diff
@@ -1,5 +1,16 @@
 List of significant changes and bug fixes going back to version 1.1.0
 
+Changes in version 3.2.3
+
+ This is a patch release for Audacity 3.2. It adds ability to share audio publicly and fixes some bugs.
+
+   * #3884 Audio can now be shared publicly on audio.com
+   * #4040 fixed the screenshot tool
```

Comment:
I wonder if always should start with a capital letter (or otherwise) for consistency sake